src/components/SubDescription.js

```diff
@@ -0,0 +1,191 @@
+import React, { useEffect, useState } from "react";
+import useDocusaurusContext from "@docusaurus/useDocusaurusContext";
+
+function SubProjectDescription({ selectedProject,issueNumber, currentIssue }) {
+  const [description, setDescription] = useState(null);
+  const { siteConfig } = useDocusaurusContext();
+  const API_BASE_URL = siteConfig.customFields.API_BASE_URL;
+  const API_AUTH_KEY = siteConfig.customFields.API_AUTH_KEY;
+
+  useEffect(() => {
+    fetch(
+      `${API_BASE_URL}/issues/${currentIssue}/${issueNumber}`,
+      {
+        method: "GET",
+        headers: {
+          "X-Secret-Key": API_AUTH_KEY,
+        },
+      },
+    )
+      .then((response) => {
+        return response.json();
+      })
+      .then((data) => {
+        setDescription(() => data);
+      })
+      .catch((error) => {
+        console.log("error", error.message);
```

Comment:
Error page handling to be added?

Reply:
will add another component for error handling responses